tests/integration/test_data/yarn_e2e_test/container/Containerfile

```diff
@@ -0,0 +1,19 @@
+FROM docker.io/node:18.19.0
+
+# Test disabled network access
+RUN if curl -IsS www.google.com; then echo "Has network access!"; exit 1; fi
+
+# Print cachi2 env vars file
+RUN cat /tmp/cachi2.env
+
+# Check yarn deps
+RUN ls /tmp/yarn_e2e_test-output/deps/yarn
+
+# Check content of source repository folder
+RUN ls /tmp/yarn_e2e_test-source/
+
+WORKDIR /tmp/yarn_e2e_test-source
+RUN . /tmp/cachi2.env && yarn install --mode skip-build && yarn build
+
+RUN cp -r /tmp/yarn_e2e_test-source /src
```

Comment:
A small improvement suggestion here (considering I don't want to touch the integration tests helper for now).
```
FROM docker.io/node:18.19.0

# Test disabled network access
RUN if curl -IsS www.google.com; then echo "Has network access!"; exit 1; fi

# Print cachi2 env vars file
RUN cat /tmp/cachi2.env

# Check yarn deps
RUN ls /tmp/yarn_e2e_test-output/deps/yarn

# Check content of source repository folder
RUN ls /tmp/yarn_e2e_test-source/

# This should be a COPY, but the sources and the Containerfile are in
# different directories
RUN cp -r /tmp/yarn_e2e_test-source /src

WORKDIR /src
RUN . /tmp/cachi2.env && yarn install --mode skip-build && yarn build

CMD ["yarn", "berryscary"]
```